package.json

```diff
@@ -45,7 +45,7 @@
     "@types/tar-stream": "1.6.0",
     "ajv": "^6.6.1",
     "bluebird": "^3.5.3",
-    "docker-progress": "^3.0.4",
+    "docker-progress": "^4.0.0",
```

Comment:
I am reasoning here that although the `docker-progress` change type was 'major' in relation to the "interface" of `docker-progress`, the change is type is 'patch' in relation to the "interface" of `resin-multibuild`. The consequence of this change to users of the `resin-multibuild` package is that some promises will reject in circumstances where they previously fulfilled, but this could be described as a "patch bug fix".